include/rocksdb/db.h

```diff
@@ -1051,6 +1061,11 @@ class DB {
     return IngestExternalFile(DefaultColumnFamily(), external_files, options);
   }
 
+  // IngestExternalFiles() will ingest files for multiple column families, and
+  // record the result atomically to the MANIFEST.
```

Comment:
As comment above, we should clearly document to behavior about whether the data becomes visible atomically.

Reply:
Done.